Tests/Acceptance/ClearExpiredTokensCommandTest.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+namespace Trikoder\Bundle\OAuth2Bundle\Tests\Acceptance;
+
+use DateTime;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Tester\CommandTester;
+use Trikoder\Bundle\OAuth2Bundle\Manager\AccessTokenManagerInterface;
+use Trikoder\Bundle\OAuth2Bundle\Manager\ClientManagerInterface;
+use Trikoder\Bundle\OAuth2Bundle\Manager\RefreshTokenManagerInterface;
+use Trikoder\Bundle\OAuth2Bundle\Manager\ScopeManagerInterface;
+use Trikoder\Bundle\OAuth2Bundle\Tests\Fixtures\FixtureFactory;
+
+final class ClearExpiredTokensCommandTest extends AbstractAcceptanceTest
+{
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        timecop_freeze(new DateTime());
+
+        FixtureFactory::initializeFixtures(
+            $this->client->getContainer()->get(ScopeManagerInterface::class),
+            $this->client->getContainer()->get(ClientManagerInterface::class),
+            $this->client->getContainer()->get(AccessTokenManagerInterface::class),
+            $this->client->getContainer()->get(RefreshTokenManagerInterface::class)
+        );
+    }
+
+    protected function tearDown(): void
+    {
+        timecop_return();
+
+        parent::tearDown();
+    }
+
+    public function testClearExpiredAccessAndRefreshTokens(): void
+    {
+        $command = $this->command();
+        $commandTester = new CommandTester($command);
+
+        $exitCode = $commandTester->execute([
+            'command' => $command->getName(),
+        ]);
+
+        $this->assertSame(0, $exitCode);
+
+        $output = $commandTester->getDisplay();
+        $this->assertStringContainsString('Cleared 1 expired access token.', $output);
```

Comment:
@HypeMC These and some other similar tests in this PR don't actually test the feature in question. They only test the string output of the command which makes them kinda useless in its current state. The test should have assertions that prove that the tokens get actually cleared properly in the database.

Reply:
@X-Coder264 Added two additional tests, currently in the acceptance tests folder, hopefully will be moved at a later time once the infrastructure is improved.